timber-lint/src/test/java/timber/lint/WrongTimberUsageDetectorTest.kt

```diff
@@ -1425,4 +1425,38 @@ class WrongTimberUsageDetectorTest {
         .run()
         .expectClean()
   }
+
+    @Test fun exceptionLoggingWithGuardedVariable() {
```

Comment:
The indent here is wrong. Hopefully Spotless/ktlint will flag. GitHub won't let me run the checks so probably gotta kick it with a rebase or a merge.